src/Campaigns/ListTable/Columns/GoalColumn.php

```diff
@@ -33,8 +33,56 @@ public function getLabel(): string
      *
      * @param Campaign $model
      */
-    public function getCellValue($model): int
+    public function getCellValue($model): string
     {
-        return $model->goal;
+        // Temp value considering only the default form associated with the campaign
+        if ($model->defaultForm()) {
+            $goal = give_goal_progress_stats($model->defaultForm()->id);
+            $goalPercentage = ('percentage' === $goal['format']) ? str_replace('%', '',
```

Comment:
I think you could just use $goal['raw_actual'] so you don't have to use `str_replace` to "clean" the value.

Reply:
For now, I'd prefer to let it this way which is the same way used on the _"All Forms"_ list table just to make sure we'll keep the consistency just in case the `$goal['raw actual']` behaviors in a weird way. Also, I'm planning to revisit it in subsequent PRs and probably not use more this function to retrieve the campaign goal data.